common/models/user.js

```diff
@@ -434,6 +441,130 @@ module.exports = function(User) {
   };
 
   /**
+   * Returns default verification options to use when calling User.verify()
+   * without the `verifyOptions` param. This is specifically useful when calling
+   * this method remotely over rest.
+   *
+   * This is the full list of possible params, with example values
+   *
+   * ```js
+   * {
+   *   type: 'email',
+   *   mailer: anEmailModel,
+   *   to: 'test@email.com',
```

Comment:
> i know. But from the existing code, to can be configured. Maybe this was introduced for testing purposes. For sake of clarity we should make it explicit in the jsdoc that to defaults to user's email if available but can be overridden for testing purposes.

I see, makes sense. Let's keep the `to` option in for backwards compatibility.

> with the existing implementation a user with no email for whom the email verification method would be triggered would result in the app crashing when asserting verifyOptions.to, it's extreme: the app should not crash on such runtime scenarios.
This should be fixed outside the scope of this PR

I agree the method should not be crashing, and I am fine with fixing that problem outside of this pull request.